packages/wrangler/src/deployment-bundle/create-worker-upload-form.ts

```diff
@@ -79,6 +79,7 @@ export type WorkerMetadataBinding =
 			class_name: string;
 			script_name?: string;
 			environment?: string;
+			namespace_id?: string;
```

Comment:
this is added to the type, but not to `createWorkerUploadForm`'s logic itself. Is `namespace_id` ever uploaded in that case? Or am I missing something?

Reply:
This is generated on the server, its a property returned by the API but I think we just never added it to the types.